src/components/Sidebar/Sidebar.minors.tsx

```diff
@@ -24,17 +26,23 @@ export function Item({
         onClick={onClick}
         size="md"
         variant="minimalTransparent"
+        isActive={isActive}
+        activeStyles={activeStyles}
         {...props}
       />
     </Tip>
   );
 }
 
-const ItemStyled = styled(Button)`
+const ItemStyled = styled(Button)<{
+  isActive?: boolean;
+  activeStyles?: ActiveStyles;
+}>`
   height: 3rem;
   text-align: left;
   padding: 0.25rem 0.25rem 0.25rem 0.75rem;
   justify-content: start;
+  ${({ isActive, activeStyles }) => (isActive ? activeStyles : '')};
```

Comment:
```suggestion
  ${({ isActive, activeStyles }) => (isActive && activeStyles ? activeStyles : '')};
```

Reply:
it checkes here https://github.com/vimeo/iris/pull/193/files#diff-9a8e7714f9d7c2c36b78900abecba14232bc218b61e5cb3181c5ac73bd0709efR68